packages/global-cli/README.md

```diff
@@ -1,126 +1,65 @@
-## Running CLI with local modifications
+## React Native Global CLI
 
-React Native is distributed as two npm packages, `react-native-cli` and `react-native`. The first one is a lightweight package that should be installed globally (`npm install -g react-native-cli`), while the second one contains the actual React Native framework code and is installed locally into your project when you run `react-native init`.
+`react-native-cli` is a command line tool for initializing React Native projects.
 
-Because `react-native init` calls `npm install react-native`, simply linking your local github clone into npm is not enough to test local changes.
+## Installation
 
-### Introducing Sinopia
+Install it as a global module:
 
-[Sinopia] is an npm registry that runs on your local machine and allows you to publish packages to it. Everything else is proxied from `npmjs.com`. We'll set up sinopia for React Native CLI development. First, install it with:
+```sh
+yarn global add react-native-cli
+```
 
-    $ npm install -g sinopia
+It will create a global `react-native` binary (even though the package name is `react-native-cli`).
 
-Now you can run sinopia by simply doing:
+## Usage
 
-    $ sinopia
+To create a new React Native Project called "AwesomeProject" you can run:
 
-Running it for the first time creates a default config file. Open `~/.config/sinopia/config.yaml` and configure it like this (note the `max_body_size`):
+```sh
+react-native init AwesomeProject
+```
 
-    storage: ./storage
+It will install `react-native`, `react`, `jest` and a bunch of other necessary packages from the [default template](https://github.com/facebook/react-native/tree/master/template).
 
-    auth:
-      htpasswd:
-        file: ./htpasswd
+After it's finished, your AwesomeProject should be ready to use. From this point, you should use your local `react-native` binary to run the proper [React Native CLI](../cli):
 
-    uplinks:
-      npmjs:
-        url: https://registry.npmjs.org/
+```sh
+yarn react-native link native-dep
```

Comment:
does the `yarn` command still needed to be used here, or was it because you used `npx` earlier in the section?

Reply:
I stand that we should encourage using local `react-native` installation (with the real CLI, not a proxy like `react-native-cli`) to always use up-to-date CLI, instead of messing with globals.